salt/resource-manager/init.sls

```diff
@@ -1,7 +1,11 @@
 {% set resource_manager_path = pillar['resource_manager']['path'] %}
 {% set map_file = resource_manager_path + "policies/map.txt" %}
 {% set policy_file_link = resource_manager_path + pillar['resource_manager']['policy_file'] %}
-{% set execs = [ 'spark-submit', 'spark-shell', 'pyspark', 'mapred', 'hive', 'beeline' ] %} 
+{% set execs = [ 'spark-submit', 'spark-shell', 'pyspark', 'mapred', 'hive', 'beeline', 'flink', 'pyflink.sh', 'start-scala-shell.sh', 'yarn-session.sh' ] %} 
+{% set flink_version = pillar['flink']['release_version'] %}
```

Comment:
could you rebase your branch with develop as you did not have flink in your pillar ATM please

Reply:
Sure, will rebase the branch.

Reply:
Done..

Reply:
Done